src/main/splash.ts

```diff
@@ -9,10 +9,19 @@ import { join } from "path";
 import { SplashProps } from "shared/browserWinProperties";
 import { VIEW_DIR } from "shared/paths";
 
-export function createSplashWindow() {
+import { Settings } from "./settings";
+
+export async function createSplashWindow() {
```

Comment:
there doesn't seem to be the need for this to be async
```suggestion
export function createSplashWindow() {
```